src/archiefvernietigingscomponent/destruction/utils.py

```diff
@@ -0,0 +1,118 @@
+from datetime import date, datetime
```

Comment:
All functions below are used to create a report. I think they deserve a separate package/file 'report'